js/ElementRendererProvider.js

```diff
@@ -410,6 +410,35 @@ export default class ElementRendererProvider {
         Utils.appendAttributesFromObject(divCarouselWrapper, config.accessibility.web);
       }
 
+      if (config.style) {
```

Comment:
@yuliyavish I think we need to extend also carousel schema to enable `style` property on it